### PR TITLE
Add real XMPPService and expand example

### PR DESCRIPTION
**Pull request by dwilson6**
Rename XmppService to MembersService, add XMPPEvents constant and add XMPPService to angular jm.xmpp module

### CODE REVIEW

**Comment by watsoncj** [comment]
Long term, In addition to relaying the events, do you think any of the methods on XmppService.XMPP should be wrapped?

**Comment by dwilson6** [comment]
For now, let's just export the XMPP module as is and we can see if it would be more helpful to wrap it

File: src/xmpp/index.coffee [hunk 1]
```diff
@@ -1,9 +1,19 @@
 'use strict'
 name = 'jm.xmpp'
-angular.module(name, []).factory 'XmppService', [
+
+xmppModule = angular.module name, []
+xmppModule.constant 'XMPPEvents', require 'jitsi-meet/service/xmpp/XMPPEvents'
```
Comment (dwilson6) [inline src/xmpp/index.coffee:5]: @watsoncj, what do you think about prefixing the angular component names with the module (so jm.xmpp.XMPPEvents)?  And if we do that, maybe put any wrapper services like MembersService in a separate module to show what is added in this project (ajm.xmpp.MembersService).  Also, I'm going to rename MembersService to MemberService

**Comment by vdice** [comment]
As @dwilson6 mentioned, MembersService has been renamed to MemberService.  In addition, the work I started in 'bootstrap-karma' now exists here, providing karma and a simple proof-of-concept spec file around MemberService.

**Comment by watsoncj** [comment]
solid work! :bread:

**Pull request merged**